src/domain/pset.ts

```diff
@@ -13,14 +18,22 @@ import {
 } from '../application/account';
 import type { WalletRepository, AppRepository, TaxiRepository } from './repository';
 import type { CoinSelection } from './transaction';
+import { computeBalances } from './transaction';
 
-function estimateScriptSigSize(type: ScriptType): number {
+function estimateScriptSigSize(type: address.ScriptType): number {
   switch (type) {
```

Comment:
Unable to make suggestion on multiple lines since there are deleted lines in between.

The final code for the switch case should be:

```
  switch (type) {
    case address.ScriptType.P2Pkh:
      return 108;
    case address.ScriptType.P2Sh:
      return 35;
    case address.ScriptType.P2Wsh:
    case address.ScriptType.P2Tr:
    case address.ScriptType.P2Wpkh:
      return 1;
    default:
      return 0;
  }